### PR TITLE
UCT/IB: Clarify the meaning of TX_CQ_MODERATION

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by yosefe** [comment]
adding @vmaffione

File: src/uct/ib/base/ib_iface.c [hunk 1]
```diff
@@ -80,7 +80,7 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    ucs_offsetof(uct_ib_iface_config_t, tx.min_sge), UCS_CONFIG_TYPE_UINT},
 
   {"TX_CQ_MODERATION", "64",
-   "Number of send WQEs for which completion is requested.",
+   "Number of work requests to post before requesting a send completion.",
```
Comment (vmaffione) [inline src/uct/ib/base/ib_iface.c]: Maybe
"Maximum number of send WQEs whose completion can be reported with a single CQ element."
?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7323/ for details (Mellanox internal link).

File: src/uct/ib/base/ib_iface.c (hunk 1 above)
Comment (yosefe) [reply src/uct/ib/base/ib_iface.c]: How about "Maximum number of send WQEs which can be posted without requesting a send completion"?

Comment (vmaffione) [reply src/uct/ib/base/ib_iface.c]: Sounds good!

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7324/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7325/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7326/ for details (Mellanox internal link).

**Pull request merged**